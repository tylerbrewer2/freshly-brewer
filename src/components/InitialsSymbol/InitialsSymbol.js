import React from 'react';
import styled from 'styled-components';

import LargeText from '../LargeText';

// Style Helpers
import MediaWidths from '../../styles/helpers/MediaWidths';

const InitialsSymbolCore = ({ className }) => {
  return (
    <div className={className}>
      <LargeText>M</LargeText>
      <LargeText>x</LargeText>
      <LargeText>T</LargeText>
    </div>
  )
}

const InitialsSymbol = styled(InitialsSymbolCore)`
  text-align: center;

	@media (${MediaWidths.mobile}) {
		display: none;
	}
`

export default InitialsSymbol;
