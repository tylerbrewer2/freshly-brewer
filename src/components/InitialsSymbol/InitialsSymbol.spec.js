import React from 'react';
import renderer from 'react-test-renderer';

import InitialsSymbol from './InitialsSymbol';

it('renders correctly', () => {
  const tree = renderer.create(
    <InitialsSymbol />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
