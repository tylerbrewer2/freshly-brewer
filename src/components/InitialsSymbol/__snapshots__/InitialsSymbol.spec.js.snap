exports[`test renders correctly 1`] = `
<div
  className="sc-bwzfXH kvZDMd">
  <div
    className="sc-bdVaJa cEaLC"
    color="black">
    M
  </div>
  <div
    className="sc-bdVaJa cEaLC"
    color="black">
    x
  </div>
  <div
    className="sc-bdVaJa cEaLC"
    color="black">
    T
  </div>
</div>
`;
